ocr.pm

```diff
@@ -47,13 +49,12 @@ sub tesseract {
     }
 
     $img->write($imgfn);
-    if (system('tesseract', $imgfn, $txtfn) == 0) {
-        $txtfn .= '.txt';
-        if (open(my $fh, '<:encoding(UTF-8)', $txtfn)) {
-            local $/;
-            $txt = <$fh>;
-            close $fh;
-        }
+    system('tesseract', $imgfn, $txtfn);
+    $txtfn .= '.txt';
+    if (open(my $fh, '<:encoding(UTF-8)', $txtfn)) {
```

Comment:
AFAIK if using autodie(:all) you don't need to check for open too. Unless you wanted only autodie(:system) which I think you wanted.